jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java

```diff
@@ -162,6 +167,13 @@
     }
   }
 
+  private static String capitalizeFirstLetter(String string) {
```

Comment:
Capitalization is probably unnecessary, would prefer we just use the value in `getLayerType()` directly.